extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeStreamExtractor.java

```diff
@@ -528,10 +526,7 @@ public StreamInfoItem getNextStream() throws ExtractionException {
                     .getArray("contents").getObject(0).getObject("compactVideoRenderer");
 
             final TimeAgoParser timeAgoParser = getTimeAgoParser();
-            StreamInfoItemsCollector collector = new StreamInfoItemsCollector(getServiceId());
-
-            collector.commit(new YoutubeStreamInfoItemExtractor(videoInfo, timeAgoParser));
-            return collector.getItems().get(0);
+            return new YoutubeStreamInfoItemExtractor(videoInfo, timeAgoParser);
```

Comment:
```suggestion
            return new YoutubeStreamInfoItemExtractor(videoInfo, getTimeAgoParser());
```
and remove the line above